src/installer/test/Assets/TestUtils/SDKLookup/dotnet.runtimeconfig.json

```diff
@@ -1,8 +1,10 @@
 {
   "runtimeOptions": {
+    /* This is a multiline comment to test that the JSON parser is correctly
+     * set up to ignore them. */
     "framework": {
-      "name": "Microsoft.NETCore.App",
-      "version": "9999.0.0"
+      "name": "Microsoft.NETCore.App", // And this is a single-line comment
+      "version": "9999.0.0"            // that should be ignored by the parser
```

Comment:
As a side note, I think the tests that use this right now don't strictly need to do the config parsing, since they are just for validating SDK lookup - I would argue they shouldn't once we have a mockhostfxr (and so the comment validation would need to be moved).